theme/static/style/style.css

```diff
@@ -377,6 +382,11 @@ pre code {
 @media screen and (max-width: 991px){
     /* For tablets: */
 
+    /* Header section */
+    .header-img {
+        margin-top: 15%;
+    }
```

Comment:
```
    .header .col-12 {
        display: flex;
        align-items: flex-end;
    }
```
with `.header .header-img { background-position: center; }` will give you a similar effect and containers will be aligned at the bottom. Still the titles kind of look ugly because our grid structure doesn't make it easier to style the titles better.

Reply:
NOTE: just update the styles, we can think about the titles later if necessary.